packages/navbar/package.json

```diff
@@ -10,22 +10,23 @@
     "access": "public"
   },
   "scripts": {
-    "build": "run-s build:js build:cjs build:css",
-    "build:cjs": "babel --root-mode upward src --out-dir dist/cjs",
+    "build": "run-s build:cjs build:esm build:css",
+    "build:cjs": "tsc --project tsconfig.build.json --module commonjs --target es5 --outDir dist/cjs",
     "build:css": "build-css --useGlamor -i dist/cjs/css/index.js",
-    "build:js": "cross-env ESM=true babel --root-mode upward src --out-dir dist/esm",
-    "build:watch": "yarn build:js -- --watch",
+    "build:esm": "tsc --project tsconfig.build.json --module es2015 --target es5 --outDir dist/esm",
+    "build:watch": "yarn build:esm -- --watch",
     "storybook": "start-storybook -p 6006",
-    "test": "jest",
+    "test": "tsc --noEmit --project ./tsconfig.json && jest",
     "test:watch": "yarn test -- --watchAll"
   },
   "gitHead": "7418883d96a76c59a99f86c2b16516ec752bf913",
+  "style": "dist/index.css",
+  "types": "dist/esm/index.d.ts",
   "dependencies": {
     "@pluralsight/ps-design-system-core": "^6.5.1",
-    "@pluralsight/ps-design-system-filter-react-props": "^1.2.1",
     "@pluralsight/ps-design-system-icon": "^19.0.5",
     "@pluralsight/ps-design-system-navitem": "^1.4.7",
-    "prop-types": "^15.7.2"
+    "@pluralsight/ps-design-system-util": "*"
```

Comment:
Are we going to '*' on internal deps?  Doesn't seem like we have entirely.  Why here?

Reply:
i just `*` is and lerna will update it with the version range during publish. the works for deps and devDeps. lerna does not do it for peerDeps

Reply:
So, if lerna takes care of setting dependencies on publish, should we just * most all the internal psds deps?  Sounds like it to me.  Maybe there's a corner case where we have more specific version requirements, and we'd set those specifically.

Reply:
lerna replaces the `*` during publish. so i just set it initially and expect lerna to do the right thing